src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,132 @@
+package storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+import exceptions.DukeException;
+import tasklist.TaskList;
+import tasks.Deadlines;
+import tasks.DukeTask;
+import tasks.Events;
+import tasks.Todo;
+
+
+public class Storage {
+
+    private final Path dataFolder = Paths.get(System.getProperty("user.dir"), "data");
+    private final Path dukeTxt = Paths.get(System.getProperty("user.dir"), "data", "duke.txt");
+    private List<DukeTask> loadFile;
+
+    public Storage() {
+        this.loadFile = new ArrayList<>();
+    }
+
+    public List<DukeTask> load() throws DukeException {
+        return parseFile(this.getFileFromPath());
+    }
+
+    public File getFileFromPath() {
+        if (!Files.exists(this.dataFolder) || !Files.exists(this.dukeTxt)) {
+            System.out.println("Oops! You don't seem to have a load file!\n"
+                    + "Creating one now!!\n");
+        }
+        try {
+            Files.createDirectory(this.dataFolder);
+            Files.createFile(this.dukeTxt);
+        } catch (IOException e) {
+            System.out.println("Nice! We found your load file!\n"
+                    + "Loading...\n");
+        }
+
+        assert(Files.exists(this.dataFolder));
+        assert(Files.exists(this.dukeTxt));
+
+        File loadData = this.dukeTxt.toFile();
+        return loadData;
+    }
+
+    /**
+     * Parses the txt Duke file
+     * @param data File to be parsed.
+     * @return A List of DukeTasks.
+     * @throws DukeException
+     */
+    public List<DukeTask> parseFile(File data) throws DukeException {
+        try {
+            Scanner reader = new Scanner(data);
+            while (reader.hasNextLine()) {
+                String line = reader.nextLine();
+                String[] lineData = line.split(" [|] ", 2);
+                String taskType = lineData[0];
+                try {
+                    switch (taskType) {
+                    case ("T"):
+                        String[] info = lineData[1].split(" [|] ", 2);
+                        if (info[0].equals("1")) {
+                            this.loadFile.add(new Todo(info[1], true));
+                        } else {
+                            this.loadFile.add(new Todo(info[1], false));
+                        }
```

Comment:
Your if-else statements follow the java coding standard. I noticed that you did this practice for several other places as well. Keep it up!